lib/pbkit/pbkit.c

```diff
@@ -2267,7 +2267,16 @@ void pb_end(DWORD *pEnd)
 #endif
 
 #ifdef DBG
-    if (pb_BeginEndPair==0) debugPrint("pb_end without a pb_start\n");
+    if (pEnd!=pb_PushNext)
+    {
+        debugPrint("pb_end: input pointer invalid or not following previous write addresses\n");
+        assert(false);
+    }
+    if (pb_BeginEndPair==0)
+    {
+        debugPrint("pb_end without a pb_start\n");
```

Comment:
nit: This should be `pb_begin` instead of `pb_start`.

It is already a bug in the existing code.
A similar bug exists in `pb_begin` which calls itself `pb_start`.

Reply:
A separate commit was filed, listed last.